doc/Language/exceptions.pod6

```diff
@@ -57,18 +58,22 @@ It's possible to handle exceptional circumstances by supplying a C<CATCH> block:
 
     # OUTPUT: «some kind of IO exception was caught!»
 
-Here, we are saying that if any exception of type C<X::IO> occurs, then the
-message C<some kind of IO exception was caught!> will be sent to I<stderr>,
-which is what C<$*ERR.say> does, getting displayed on whatever constitutes the
-standard error device in that moment, which will probably be the console by
-default.
+Here, we are saying that if any exception of type C<X::IO> occurs, then
+the message C<some kind of IO exception was caught!> will be sent to
+I<stderr>, which is what C<$*ERR.say> does, getting displayed on
+whatever constitutes the standard error device in that moment, which
+will probably be the console by default.
 
-A X<C<CATCH>|CATCH> block uses smartmatching similar to how C<given/when>
-smartmatches on options, thus it's possible to catch and handle various
-categories of exceptions inside a C<when> block.
+Note that the match target is a role. To allow user defined exceptions
+to match in the same manner, they must implement the given role. Just
+existing in the same namespace will look alike but won't match in a
+C<CATCH> block.
 
-To handle all exceptions, use a C<default> statement. This example prints out
-almost the same information as the normal backtrace printer.
+A X<C<CATCH>|CATCH> block places any exception thrown in its topic
```

Comment:
The anchor to `CATCH` will be right here. Are you sure this is the best place? Or wouldn't it better to use CATCH block for indexing?

Reply:
I think this belongs more on the phasers page.

Reply:
Then it's better if you move the X thingie